docs/commands/super.md

```diff
@@ -399,12 +400,39 @@ be used with any output format.
 
 #### Simplified Text Outputs
 
-The `text` and `table` formats simplify data to fit within the
-limitations of text-based output. Because they do not capture all the
-information required to reconstruct the original data, they are not supported
-input formats. They may be a good fit for use with other text-based shell
+The `line`, `text`, and `table` formats simplify data to fit within the
+limitations of text-based output. They may be a good fit for use with other text-based shell
```

Comment:
I decided to keep the details for all three of these formats still in one section but gave `line` top billing because it comes with the fewest caveats.